doc/lsst.ap.verify/datasets-creation.rst

```diff
@@ -28,12 +28,26 @@ Organizing the Data
 
 * The :file:`raw` and :file:`calib` directories contain science and calibration data, respectively.
   The directories may have any internal structure.
-* The :file:`templates` directory contains an :ref:`LSST Butler repository<butler>` containing processed images useable as templates.
+* The :file:`templates` directory contains an :ref:`LSST Butler repository<butler>` containing processed images usable as templates.
```

Comment:
FUN FACT: both usable and useable are acceptable spellings! (I don't have a preference 😆 ) https://english.stackexchange.com/questions/69709/is-useable-preferred-in-certain-regions-or-just-an-alternate-spelling-of-usa

Reply:
Well, my spellchecker disagrees. :slightly_smiling_face: